cpp/inc/bond/ext/grpc/wait_callback.h

```diff
@@ -33,13 +33,11 @@ template <typename Response>
 class wait_callback final
 {
 public:
-    using arg_type = unary_call_result<Response>;
```

Comment:
Removing this is fine. Breaking change for the changelog, though.

Reply:
Sent #899 for that and closing this one.